lib/server/proxyClientRequestToApi.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+var Backbone = require("../application/Backbone");
+var Configure = require("../application/Configure");
+
+function proxyClientRequestToApi(apiHost){
+    return function(req,response,next){
+        if (!!req.headers['brisket-client']){
+            Configure.backbone();
+            Backbone.ajax( apiHost + req.path)
+            .then(function(data) {
```

Comment:
I agree with Eric that we should go as close to native as possible. That way, we have guaranteed best-we-can-get performance without overhead. 

Another concern to use `Backbone.ajax` is the fact that we have to call  `Configure.backbone();` beforehand. I don't think it's supposed to be called multiple times (once on server initialization, once here if the config is true). According to the code,
```javascript
function wrapAllAjaxCallsInPromiseSoThatAnyErrorWillBeCaught() {
    Backbone.ajax = Promise.method(Backbone.ajax);
}
``` 
`Backbone.ajax` seems to be wrapped again when Backbone is re-configured.